README.md

```diff
@@ -45,6 +46,7 @@ apiVersion: zalando.org/v1
 kind: ElasticsearchDataSet
 spec:
   replicas: 2
+  hpa_replicas: 1
```

Comment:
It's convention that Kubenetes API fields are always defined as camelCase, so the field name should ideally be `hpaReplicas`.